interactions.go

```diff
@@ -35,12 +35,14 @@ type ApplicationCommand struct {
 	Version           string                 `json:"version,omitempty"`
 	Type              ApplicationCommandType `json:"type,omitempty"`
 	Name              string                 `json:"name"`
+	NameLocalizations *map[Locale]string     `json:"name_localizations,omitempty"`
 	DefaultPermission *bool                  `json:"default_permission,omitempty"`
 
 	// NOTE: Chat commands only. Otherwise it mustn't be set.
 
-	Description string                      `json:"description,omitempty"`
-	Options     []*ApplicationCommandOption `json:"options"`
+	Description              string                      `json:"description,omitempty"`
+	DescriptionLocalizations *map[Locale]string          `json:"description_localizations,omitempty"`
+	Options                  []*ApplicationCommandOption `json:"options"`
```

Comment:
```suggestion
	Options                  []*ApplicationCommandOption `json:"options,omitempty"`
```

Reply:
While this would make sense, I'm not sure that this would be the best solution.
As with `DescriptionLocalizations,` I'm going to do a PR on that.